processors/process_lq_calibration_cut.jl

```diff
@@ -142,28 +142,64 @@ function process_lq_calibration_cut(processing_config::PropDict, l200::LegendDat
                     throw(ErrorException("Error in energy correction and normalization: $e"))
                 end
 
-                @debug "Calibrate $lq_type"
+                @debug "Drift time correction of $lq_type"
                 drift_result, drift_report = nothing, nothing
                 try 
                     drift_result, drift_report = lq_ctc_correction(lq_e_corr, dt_eff, e_cal, dep_µ, dep_σ;
                         ctc_dep_edgesigma, ctc_lq_precut_relative_cut, lq_outlier_sigma, ctc_driftime_cutoff_method, dt_eff_outlier_sigma, lq_e_corr_expression, dt_eff_expression, ctc_dt_eff_low_quantile, ctc_dt_eff_high_quantile, pol_fit_order, uncertainty=ctc_uncertainty)
                 catch e
-                    @error "Error in drift time correction: $e"
-                    throw(ErrorException("Error in drift time correction: $e"))
+                    @error "Error in drift time correction: $(truncate_error(e))"
+                    throw(ErrorException("Error in drift time correction: $(truncate_error(e))"))
                 end
 
-                #create and save plots
+                # create and save plots
                 p = LegendMakie.lplot(drift_report, e_cal, dt_eff, lq_e_corr, :DEP, title = get_plottitle(filekey, det, "LQ"), figsize = (620,400))
                 savelfig(LegendMakie.lsavefig, p, l200, filekey, det, Symbol("lq_ctc_DEP_$lq_type"))
 
                 p = LegendMakie.lplot(drift_report, e_cal, dt_eff, lq_e_corr, :whole, title = get_plottitle(filekey, det, "LQ"), figsize = (620,400))
                 savelfig(LegendMakie.lsavefig, p, l200, filekey, det, Symbol("lq_ctc_$lq_type"))
 
-                #create log entry
+
+                # create lq normalization                
+                lq_ctc = nothing
+                lq_class_expression = nothing
+                result, report = nothing, nothing
+                lq_class = nothing
+                try
+                    lq_class_expression = drift_result.func
```

Comment:
No need to have the `lq_class_expression` inside the `try` because it depends on `drift_result`:
```suggestion
                # create lq normalization                
                lq_class_expression = drift_result.func
                lq_ctc = nothing
                result, report = nothing, nothing
                lq_class = nothing
                try
```